appalerts/templates/appalerts/alert_list.html

```diff
@@ -103,6 +129,13 @@ <h2><i class="iconify" data-icon="mdi:alert-octagram"></i> App Alerts</h2>
        style="{% if object_list.count > 0 %}display: none;{% endif %}">
     There are currently no alerts for you.
   </div>
+
+{% if dismissed %}
+  {% if object_list.count > pg_number %}
+    {% include 'projectroles/_pagination.html' with pg_small=False %}
```

Comment:
Pagination doesn't work, the page selection is not visible even if I've created more dismissed alerts than could fit on the next page.